### PR TITLE
Handle Multiple DOIs from manifest

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

**Comment by maxulysse** [comment]
cf https://github.com/nf-core/sarek/pull/1494

File: nf_core/pipeline-template/subworkflows/local/utils_nfcore_pipeline_pipeline/main.nf [hunk 1]
```diff
@@ -243,8 +243,14 @@ def methodsDescriptionText(mqc_methods_yaml) {
...
-    meta["doi_text"] = meta.manifest_map.doi ? "(doi: <a href=\'https://doi.org/${meta.manifest_map.doi}\'>${meta.manifest_map.doi}</a>)" : ""
-    meta["nodoi_text"] = meta.manifest_map.doi ? "": "<li>If available, make sure to update the text to include the Zenodo DOI of version of the pipeline used. </li>"
+    if (meta.manifest_map.doi) {
+        def temp_doi_ref = ""
+        String[] manifest_doi = meta.manifest_map.doi.split(",")
+        for (String doi_ref: manifest_doi) temp_doi_ref += "(doi: <a href=\'https://doi.org/${doi_ref}\'>${doi_ref}</a>), "
+        meta["doi_text"] = temp_doi_ref.substring(0, temp_doi_ref.length()-2)
+    } else {
+        meta["nodoi_text"] = "<li>If available, make sure to update the text to include the Zenodo DOI of version of the pipeline used. </li>"
+    }
```
Comment (maxulysse) [inline nf_core/pipeline-template/subworkflows/local/utils_nfcore_pipeline_pipeline/main.nf:253]:
```suggestion
    if (meta.manifest_map.doi) {
        def temp_doi_ref = ""
        String[] manifest_doi = meta.manifest_map.doi.split(",")
        for (String doi_ref: manifest_doi) temp_doi_ref += "(doi: <a href=\'https://doi.org/${doi_ref}\'>${doi_ref}</a>), "
        meta["doi_text"] = temp_doi_ref.substring(0, temp_doi_ref.length()-2)
    } else meta["doi_text"] = ""
    meta["nodoi_text"] = meta.manifest_map.doi ? "" : "<li>If available, make sure to update the text to include the Zenodo DOI of version of the pipeline used. </li>"
```

**Review by mirpedrol — APPROVED** [review]
LGTM, don't forget to update the CHANGELOG.md 🙂

**Comment by maxulysse** [comment]
> LGTM, don't forget to update the CHANGELOG.md 🙂

Need to had another fix in

**Comment by mashehu** [comment]
Fix for CI tests is in #2949

**Pull request merged**